src/components/Header/SubHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { useTheme } from '@mui/material/styles';
+import { LogoPagoPAProduct } from '@pagopa/mui-italia';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import { ArcRoutes } from 'routes/routes';
+import Typography from '@mui/material/Typography';
+import { useStore } from 'store/GlobalStore';
+import { useCartActions } from 'store/CartStore';
+
+export const SubHeader = () => {
+  const { spacing } = useTheme();
+
+  const {
+    state: { cart }
+  } = useStore();
+
+  const { toggleCartDrawer } = useCartActions();
+
+  const Cart = () => {
+    return (
+      <Button variant="naked" sx={{ gap: 1 }} onClick={toggleCartDrawer} name="CartButton">
```

Comment:
Please add a proper title or aria-label to identify the action trigged by this button

Reply:
I've refactored this component to improve a11y in the latest commit